iavl_tree.go

```diff
@@ -43,6 +38,9 @@ func (t *IAVLTree) String() string {
 	return "IAVLTree{" + strings.Join(leaves, ", ") + "}"
 }
 
+// DEPRECATED.
```

Comment:
Can we note what it's been replaced with

Reply:
Yup. 27b5237